packages/components/src/color-picker/inputs.js

```diff
@@ -103,7 +103,7 @@ export class Inputs extends Component {
 		} else if ( this.state.view === 'rgb' ) {
 			this.setState( { view: 'hsl' } );
 
-			speak( __( 'Hue/saturation/lightness mode active' ) );
+			speak( __( 'HSL mode active' ) );
```

Comment:
Given that text is spoken aloud to users with screen readers, `Hue/saturation/lightness` might be way better to understand for them than `HSL`.

Reply:
`HSL` is a not a new word, we already use it in other strings. This PR is just for consistency - to use the same work in other places.

As for screen readers - just as regular users understand the meaning from the context, so are screen reader users.